src/tsserver/webServer.ts

```diff
@@ -49,17 +52,34 @@ namespace ts.server {
     function createWebSystem(args: string[]) {
         Debug.assert(ts.sys === undefined);
         const returnEmptyString = () => "";
+
         const sys: ServerHost = {
             args,
-            newLine: "\r\n", // TODO::
-            useCaseSensitiveFileNames: false, // TODO::
-            readFile: returnUndefined, // TODO:: read lib Files
+            newLine: "\n", // This can be configured by clients

```

Comment:
i think "\r\n" is the default that compiler uses so seems like we should stick to that.
https://github.com/microsoft/TypeScript/blob/master/src/compiler/utilities.ts#L5005